yeti/api.py

```diff
@@ -21,6 +21,38 @@
 API_TOKEN_ENDPOINT = "/api/v2/auth/api-token"
 
 
+SUPPORTED_IOC_TYPES = [
```

Comment:
Do you think we should add a dedicated api endpoint in yeti observables definitions to provide supported types? Otherwise, how would you handle custom observables?

Reply:
Yes, that's a great point. Opened https://github.com/yeti-platform/yeti/issues/1254 to track this.